gammapy/background/off_data_background_maker.py

```diff
@@ -21,28 +23,30 @@
 class OffDataBackgroundMaker(object):
     """OffDataBackgroundMaker class.
 
-    Class that will select an OFF list run from a Data list and then group this runlist in group of
-    zenithal angle and efficiency. Then for each group, it will compute the background rate model in
-    3D *(X, Y, energy)* or 2D *(energy, offset)* via the class `~gammapy.background.CubeBackgroundModel` (3D) or
-    `~gammapy.background.EnergyOffsetBackgroundModel` (2D).
-
-    Parameters
-    ----------
-    data_store : `~gammapy.data.DataStore`
-        Data for the background model
-    run_list : str
-        filename where is store the OFF run list
-    outdir : str
-        directory where will go the output
-    obs_table : `~astropy.table.Table`
-        observation table of the OFF run List used for the background modelling
-        require GROUP_ID column
-    excluded_sources : `~astropy.table.Table`
-        Table of excluded sources.
-        Required columns: RA, DEC, Radius
-    """
-
-    def __init__(self, data_store, outdir=None, run_list=None, obs_table=None, excluded_sources=None):
+        Class that will select an OFF list run from a Data list and then group this runlist in group of
+        zenithal angle and efficiency. Then for each group, it will compute the background rate model in
+        3D *(X, Y, energy)* or 2D *(energy, offset)* via the class `~gammapy.background.CubeBackgroundModel` (3D) or
+        `~gammapy.background.EnergyOffsetBackgroundModel` (2D).
+
+        Parameters
+        ----------
+        data_store : `~gammapy.data.DataStore`
+            Data for the background model
+        run_list : str
+            filename where is store the OFF run list
+        outdir : str
+            directory where will go the output
+        obs_table :gamm `~astropy.table.Table`
+            observation table of the OFF run List used for the background modelling
+            require GROUP_ID column
+        ntot_group: int
```

Comment:
There has to be a space before the colon in the docstring, or it will not be rendered correctly by Sphinx.
I.e. put it like this:
```
ntot_group : int
```